lib/tx_build/host_function.ex

```diff
@@ -108,4 +124,16 @@ defmodule Stellar.TxBuild.HostFunction do
     host_function_type = HostFunctionType.new()
     HostFunctionXDR.new(sc_vec, host_function_type)
   end
+
+  def to_xdr(%__MODULE__{
+        type: :install,
+        code: code
+      }) do
+    host_function_type = HostFunctionType.new(:HOST_FUNCTION_TYPE_INSTALL_CONTRACT_CODE)
+
+    code
+    |> InstallContractCodeArgs.new()
+    |> InstallContractCodeArgs.to_xdr()
+    |> HostFunctionXDR.new(host_function_type)
```

Comment:
I think you can avoid creating the `Stellar.TxBuild.InstallContractCodeArgs` module, IMO it doesn't add much value or functionality.
 
You can do directly in `HostFunction.to_xdr/1`

```elixir
code
|> VariableOpaque256000.new()
|> InstallContractCodeArgs.new() # it refers to StellarBase
|> HostFunctionXDR.new(host_function_type)
```